### PR TITLE
Implements UK Payment Initiation support in Python

### PR DESCRIPTION
**Pull request by skansal-plaid**
Implements UK Payment Initiation support in Python.

Tested locally that both payment-initiation and the default flow work successfully.

### CODE REVIEW

File: python/templates/index.ejs [hunk 1]
```diff
@@ -40,6 +41,11 @@
               <h3 class="item-overview__heading">access_token</h3>
               <p class="item-overview__id" id="access_token">••••••••hsakjsl</p>
             </div>
+            <!-- The payment_token is only relevant for the UK Payment Initiation product. -->
+            <div class="payment_initiation item-overview__column" style='display:none;'>
+              <h3 class="item-overview__heading">payment_token</h3>
+              <p class="item-overview__id" id="payment_token">••••••••hsakjsl</p>
```
Comment (jasonjia93) [inline python/templates/index.ejs]: Should this be a different example token than the access token from above?

File: python/templates/index.ejs [hunk 2]
```diff
@@ -235,27 +262,69 @@
...
+          env: '{{ plaid_environment }}',
+          product: products,
+          key: '{{ plaid_public_key }}',
+          countryCodes: '{{ plaid_country_codes }}'.split(','),
+          paymentToken: paymentToken,
+          language: 'en',
+          onSuccess: function(public_token) {
+            // This public token exchange step is not relevant for the
+            // payment_initiation product and should be skipped.
+            $.post('/get_access_token', {
```
Comment (jasonjia93) [inline python/templates/index.ejs]: It looks like this call to `$.post('/get_access_token'` isn't actually needed based on the comment?

Comment (skansal-plaid) [reply python/templates/index.ejs]: Yes but it is required for the success page to show item id, access token, etc so I did not want to break that.

File: python/templates/index.ejs (hunk 1 above)
Comment (skansal-plaid) [reply python/templates/index.ejs]: The shape of both tokens is the same, and this value of the payment token only shows up when the `payment_initiation` product is used, in which case this value is substituted with the value of the generated payment token. So I believe this HTML should be good as is.

Note: Payment Token looks like `payment-token-sandbox-00a0000a-aaaa-0000-aa00-00000aaa00a0`. Access token looks like `access-sandbox-aa000000-00a0-0000-000a-a0000000a000`.

**Pull request merged**